tfjs-core/src/util_test.ts

```diff
@@ -665,3 +665,58 @@ describe('util.decodeString', () => {
     expect(util.isPromise(promise3)).toBeFalsy();
   });
 });
+
+describe('setTimeout', () => {
+  // If we set a larger number here, an error will be reported as "'expect'
+  // was used when there was no current spec, this could be because an
+  // asynchronous test timed out".
+  const totalCount = 8;
+  const skipCount = 5;
+
+  it('setTimeout', () => {
+    let count = 0;
+    let startTime = performance.now();
+    let totalTime = 0;
+    setTimeout(_testSetTimeout, 0);
+
+    function _testSetTimeout() {
+      let endTime = performance.now();
+      count++;
+      if (count > skipCount) {
+        totalTime += endTime - startTime;
+      }
+      if (count === totalCount) {
+        let averageTime = totalTime / (totalCount - skipCount);
+        console.log(`averageTime of setTimeout is ${averageTime} ms`);
+        // We don't have expect here as in some browsers, like Chrome Canary,
+        // nesting level threshold is set to 100 instead of 5.
```

Comment:
With the fix for totalCount above, you might be able to increase the count above 100 and add an `expect`. Otherwise, it may be best to remove this test or mark it with `xit`, since it doesn't appear to be testing anything (unless you're testing that no errors are thrown. If so, you should call `expect().nothing()`).

Reply:
Increased to 100